src/Section.js

```diff
@@ -2,6 +2,9 @@ import React, { PropTypes } from 'react';
 import { marg, pad } from './styles';
 
 const Section = React.createClass({
+/**
+ * Section is a layout primitive.
+ */
```

Comment:
Good catch, these components with one liner descriptions aren't exposed in the guide yet. I think they will be documented in a layout section. I'm not sold on the best way to do these yet. Seems one component that one can pass `header`, `section`, `div`,  or `span` makes more sense.